app/views/notifications/v3m0/revocation_reasons.scala.html

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import include._
+@import models.notifications.NotificationParams
+@import config.{ ApplicationConfig}
+
+@(notificationParams: NotificationParams)(implicit request: Request[_],m:Messages, lang: Lang, appConfig: ApplicationConfig)
+
+@header = {
+    @components.back_link()
+    <header class="page-header">
+        <h1 class="heading-xlarge">Your supervision has been revoked</h1>
+        <p class="heading-secondary"><span class="visuallyhidden">This section is </span>Your registration</p>
+    </header>
+}
+
+@main(
+    title = "Your supervision has been revoked - Your registration",
+    heading = header
+)  {
+
+    @notificationParams.amlsRefNo.map{ amlsRefNo =>
+        <p>Your reference: @amlsRefNo</p>
+    }
+
+    <p>Your supervision under The Money Laundering, Terrorist Financing and Transfer of Funds (Information on the Payer) Regulations 2017 for @{notificationParams.businessName.map{d => d}} was revoked on @{notificationParams.endDate.map{d => d}}.</p>
+    <p>This means you are no longer registered for supervision with HMRC. You need to be +registered with a supervisory body+ if the Money Laundering Regulations apply to your business. You may be subject to civil sanctions or criminal proceedings if you continue to trade in activities covered by the Money Laundering, Terrorist Finance and Transfer of Funds (Information on the Payer) Regulations 2017.<p>
```

Comment:
will + underline the text: registered with a supervisory body ? just asking

Reply:
No good spot I have changed this